extensions/classifiers/text-input.tokenizer.spec.ts

```diff
@@ -19,23 +19,24 @@
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // the code corresponding to the spec is upgraded to Angular 8.
 import { UpgradedServices } from 'services/UpgradedServices';
+import { TextInputTokenizer } from 'classifiers/text-input.tokenizer';
 // ^^^ This block is to be removed.
 
 describe('Text Input tokenizer', function() {
   beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
+  beforeEach(angular.mock.module('oppia', ($provide) => {
```

Comment:
Once a service is migrated and its spec file -- we don't use the UpgradedServices for that file. The reason is UpgradedServices are used only for those services that depend on migrated services but are themselves not migrated.

Here you have migrated the text-input.tokenizer so we don't need anything from UpgradedServices now. Let me know if you have any questions on this.

Reply:
Yeah, I understand.
I was thinking of this as well.